README.md

```diff
@@ -1,22 +1,120 @@
 # PartBot
 
-_but better_
+_A Modern, Modular Pokemon Showdown Bot, Reimagined from the Ground Up_
 
 ---
 
-Hiya! This is an ongoing effort to rewrite [PartBot](https://github.com/PartMan7/PartBot) from the bottom-up, with
-better practices than the steaming pile of bamboo sticks that currently supports it.
+Welcome to **PartBot**, a next- generation chatbot and automation framework built with modern engineering standards and TypeScript. This project is an ongoing, comprehensive rewrite of the original repository [Part Professor](https://github.com/PartMan7/PartProfessor), designed to be more maintainable, extensible, and robust leaving behind legacy practices for a cleaner, scalable future.
 
-This might take a while...
+---
+
+## Table of Contents
+
+- [Project Overview](#project-overview)
+- [Features](#features)
+- [Architecture](#architecture)
+- [Getting Started](#getting-started)
+- [Development Guide](#development-guide)
+- [Planned Features & Roadmap](#planned-features--roadmap)
+- [Known Issues](#known-issues)
+- [Feedback & Contributions](#feedback--contributions)
+- [Credits](#credits)
+- [License](#license)
+
+---
+
+## Project Overview
+
+**PartBot** aims to provide a powerful and flexible platform for running bots on [Pokemon Showdown](https://pokemonshowdown.com/) and similar systems. The codebase is being rebuilt from scratch to ensure best practices, clear structure, and a solid foundation for future growth. This repository is actively developed and welcomes contributions from the community.
+
+## Features
+
+- **TypeScript-first:** Strongly typed codebase for reliability and maintainability.
+- **Modular Design:** Add, remove, or update components with minimal friction.
+- **Pokémon Showdown Integration:** Automated room management, moderation, and custom features.
+- **Web Module:** Built-in CDN and file hosting, deployable via NGINX with configurable scripts.
+- **Extensible Plugin System:** (Planned) Easily develop and plug in new features or commands.
+- **Community Driven:** Open to suggestions, feedback, and pull requests.
+
+## Architecture
+
+- **Core:** The main logic for bot operations, communication, and orchestration.
```

Comment:
There is no 'core'; the closest is `ps/commands/games/core.tsx` which is the main games commands file